railgun/test/performance/perf_ingress_test.go

```diff
@@ -0,0 +1,77 @@
+//+build performance_tests
```

Comment:
Yes, I think it could make some sense to separate these out into their own build tag and directory for now, that will help isolation and navigation while we continue to grow these tests :+1: